user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -27,3 +29,81 @@ Please note the following regarding suggestion events:
+
+1. Verify that the ![Feedback](~/user-guide/images/Feedback_Column.png) column is visible. If it is not:
+
+   - Close the *Anomalies* or *Suggestion events* tab and reopen it. However, make sure not to use the *Reopen closed alarm lists* option. See [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+
+     This step might be necessary if you have recently upgraded from a version of DataMiner that did not include the Feedback column to one that does (DataMiner 10.4.11/10.5.0 or higher). Since Cube saves your open tabs along with their column layout, the new column will not automatically appear in tabs that were open before the upgrade.
+
+   - Ensure you are using DataMiner version 10.4.11/10.5.0 or higher, or confirm that the *AnomalyFeedback* option is included in the *SoftLaunchOptions.xml* file.
+
+   - If possible, manually add the column by right-clicking the column headers and selecting *Add or remove columns* > *Actions* > *Feedback*.
+
+1. Hover the mouse pointer over a suggestion event or alarm to view the ![Thumbs-up](~/user-guide/images/Thumbs_Up.png) and ![Thumbs-down](~/user-guide/images/Thumbs_Down.png) icons.
+
+   - If you approve of the anomaly and want to receive similar ones in the future, click the thumbs-up icon in the feedback column. Alternatively, you can right-click the suggestion event/alarm and select *Like*.
+
+   - If you disapprove of the anomaly and do not wish to be notified of similar events in the future, click the thumbs-down icon. Alternatively, you can right-click the suggestion event/alarm and select *Dislike*.
+
+   Once selected, the chosen icon will be pinned in the column as a black icon. If you change your mind, you can click the other option, and only the final feedback will be registered<!--RN 39082-->.
+
+   > [!NOTE]
+   >
+   > - The ![Thumbs-up](~/user-guide/images/Thumbs_Up.png) or ![Thumbs-down](~/user-guide/images/Thumbs_Down.png) icons will only appear for alarms or suggestion events generated by the behavioral anomaly detection feature.
+   > - You can only see the feedback icons you have clicked yourself. Feedback from other users is not visible.
+
+The feedback you provide will be taken into account in the future.
+
+Feedback on active alarms is saved in the user settings and will re-appear after closing and reopening Cube. Feedback on history alarms will disappear after restarting Cube, but it will still be used by SLAnalytics to improve the detection of future anomalies.
+
+### Follow-up actions after providing feedback
+
+After providing feedback on a suggestion event or alarm, a light bulb icon may appear next to the ![Feedback](~/user-guide/images/Feedback_Column.png) column. Clicking this icon will reveal a follow-up action based on your feedback<!--RN 39809 + 39640 + 39480-->. Alternatively, you can right-click the suggestion event or alarm and select the follow-up action from the context menu.
```

Comment:
If this only appears for the user who has provided the feedback, I'd suggest to adjust this as follows:
```suggestion
After you have provided feedback on a suggestion event or alarm, a light bulb icon may appear next to the ![Feedback](~/user-guide/images/Feedback_Column.png) column. Clicking this icon will reveal a follow-up action based on your feedback<!--RN 39809 + 39640 + 39480-->. Alternatively, you can right-click the suggestion event or alarm and select the follow-up action from the context menu.
```